### PR TITLE
i18n(es): Update `markdoc`, `mdx` & `vue`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `markdoc`, `mdx` & `vue` based on this commit 3a466f740bdf9375d57122e8cba4344467027912

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/markdoc.mdx [hunk 1]
```diff
@@ -486,6 +486,40 @@ Para lograr una experiencia más similar a Markdown, donde los elementos HTML pu
...
+    // ...
++   integrations: [markdoc({ ignoreIndentation: true })],
+    //                       ^^^^^^^^^^^^^^^^^^^^^^^
+  });
+```
+
+```md
+# Bienvenido a Markdoc con etiquetas indentadas 👋
+
+# Note: Can use either spaces or tabs for indentation
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/markdoc.mdx:510]:
```suggestion
# Nota: Puedes usar tanto espacios como tabulaciones para la identación
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good work 🚀!

**Pull request merged**